flink-streaming-connectors/flink-connector-elasticsearch2/src/main/java/org/apache/flink/streaming/connectors/elasticsearch2/ElasticsearchSink.java

```diff
@@ -244,12 +260,7 @@ public void close() {
 		}
 
 		if (hasFailure.get()) {
-			Throwable cause = failureThrowable.get();
-			if (cause != null) {
-				throw new RuntimeException("An error occured in ElasticsearchSink.", cause);
-			} else {
-				throw new RuntimeException("An error occured in ElasticsearchSink.");
-			}
+			LOG.error("Some documents failed while indexing to Elasticsearch: " + failureThrowable.get());
```

Comment:
Do you mean to substitute the current line 263 with a LOG.DEBUG + stacktrace or leave the current line or to add another LOG.debug line where we log only the stacktrace? Currently the detail of the error is logged within afterBulk(). Should we modify that part?

How do you suggest to control the error behaviour ? Is it ok comething like 

public static final String CONFIG_KEY_BULK_ERROR_POLICY = "bulk.error.policy";

that a user can set to "strict" or "lenient" in the userConfig object?